com.roundabout_host.Izvor.yaml

```diff
@@ -0,0 +1,35 @@
+id: com.roundabout_host.Izvor
```

Comment:
Do you control https://roundabout-host.com/ or is that a codehost like github outside your control?

Reply:
this question was not answered but you just resolved it.

see also https://github.com/flathub/flathub/pull/5957#discussion_r1897422924